src/main/webapp/style.css

```diff
@@ -0,0 +1,13 @@
+/* Copyright 2020 Google LLC
```

Comment:
This file will be just a source of conflicts, as Olga uploads one that has some actual code.
Consider not uploading this file (you'll need to change index.html accordingly for the time being).